src/index.html

```diff
@@ -9,10 +9,62 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/stars.css"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/stars__star.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <body>
+      <section class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--1">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--2">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--3">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--5">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </section>
```

Comment:
It's great that you're using semantic tags such as `section`, but the content inside these sections are not semantically clear. Consider adding some descriptive text or alternative content to give more context about these sections. If the `div` elements are meant to be visual representations, consider using `span` with appropriate classes or ids.